### PR TITLE
Mod: Remove unnecessary return and judgement.

### PR DESCRIPTION
**Pull request by YongHaoWu**
**What this PR does**: Sharp the code.

**Which issue(s) this PR fixes**: Readable code.

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: common/rpc_service.go [hunk 1]
```diff
@@ -24,14 +24,9 @@ import (
...
 	"unicode/utf8"
-)
 
-import (
-	perrors "github.com/pkg/errors"
-)
-
-import (
 	"github.com/apache/dubbo-go/common/logger"
+	perrors "github.com/pkg/errors"
```
Comment (AlexStocks) [inline common/rpc_service.go:29]: hey, guy, pls do not change our import code format. recover it pls.

File: config/base_config.go [hunk 2]
```diff
@@ -22,15 +22,10 @@ import (
...
-)
-
-import (
-	perrors "github.com/pkg/errors"
-)
 
-import (
 	"github.com/apache/dubbo-go/common/config"
 	"github.com/apache/dubbo-go/common/logger"
+	perrors "github.com/pkg/errors"
```
Comment (AlexStocks) [inline config/base_config.go:28]: recover it pls.

File: config/reference_config.go [hunk 3]
```diff
@@ -23,20 +23,16 @@ import (
...
-)
 
-import (
 	"github.com/apache/dubbo-go/cluster/directory"
 	"github.com/apache/dubbo-go/common"
 	"github.com/apache/dubbo-go/common/constant"
 	"github.com/apache/dubbo-go/common/extension"
 	"github.com/apache/dubbo-go/common/proxy"
 	"github.com/apache/dubbo-go/protocol"
+	gxstrings "github.com/dubbogo/gost/strings"
```
Comment (AlexStocks) [inline config/reference_config.go:35]: recove it.

File: registry/protocol/protocol.go [hunk 4]
```diff
@@ -21,23 +21,21 @@ import (
...
 
-import (
 	"github.com/apache/dubbo-go/common"
 	"github.com/apache/dubbo-go/common/constant"
 	"github.com/apache/dubbo-go/common/extension"
 	"github.com/apache/dubbo-go/common/logger"
 	"github.com/apache/dubbo-go/common/proxy/proxy_factory"
 	"github.com/apache/dubbo-go/config"
 	"github.com/apache/dubbo-go/config_center"
+	gxset "github.com/dubbogo/gost/container/set"
```
Comment (AlexStocks) [inline registry/protocol/protocol.go:32]: recover it.

File: remoting/zookeeper/curator_discovery/service_discovery.go [hunk 5]
```diff
@@ -22,18 +22,14 @@ import (
...
 	"github.com/dubbogo/go-zookeeper/zk"
-	perrors "github.com/pkg/errors"
-)
 
-import (
 	"github.com/apache/dubbo-go/common/constant"
 	"github.com/apache/dubbo-go/common/logger"
 	"github.com/apache/dubbo-go/remoting"
 	"github.com/apache/dubbo-go/remoting/zookeeper"
+	perrors "github.com/pkg/errors"
```
Comment (AlexStocks) [inline remoting/zookeeper/curator_discovery/service_discovery.go:32]: recover it.

**Review by zouyx — COMMENTED** [review]
I think more improve as below
* change target branch to `develop` 
* fix travis
* change PR title, base on https://github.com/apache/dubbo-go/blob/master/contributing.md

File: common/rpc_service.go (hunk 1 above)
Comment (YongHaoWu) [reply common/rpc_service.go:29]: @AlexStocks Hi, this is `vim-go` 's behavior.  I think it 'll change  `import order` sorted by alphabet. Can you accept this?

Comment (AlexStocks) [reply common/rpc_service.go:29]: > @AlexStocks Hi, this is `vim-go` 's behavior. I think it 'll change `import order` sorted by alphabet. Can you accept this?

no.  we dubbo-go all code files follow the same import code style.

Comment (YongHaoWu) [reply common/rpc_service.go:29]: @AlexStocks Sure, I will keep it as the same. ps, Could u share the  `import code style`?

**Comment by YongHaoWu** [comment]
@zouyx hi, about the `travis` error, it is strange that my commit base on `develop` branch will pass( it was based on `master` branch before). Is it the reason why it reported error?

File: common/rpc_service.go (hunk 1 above)
Comment (AlexStocks) [reply common/rpc_service.go:29]: refer to https://github.com/apache/dubbo-go/blob/develop/contributing.md

**Comment by zouyx** [comment]
> @zouyx hi, about the `travis` error, it is strange that my commit base on `develop` branch will pass( it was based on `master` branch before). Is it the reason why it reported error?

Maybe.........

File: config/base_config.go (hunk 2 above)
Comment (zouyx) [inline config/base_config.go:28]: Still wrong style

File: config/reference_config.go (hunk 3 above)
Comment (zouyx) [inline config/reference_config.go:35]: still wrong style

File: filter/filter_impl/access_log_filter.go [hunk 6]
```diff
@@ -23,9 +23,7 @@ import (
 	"reflect"
 	"strings"
 	"time"
-)
```
Comment (zouyx) [inline filter/filter_impl/access_log_filter.go:26]: recover it

File: protocol/rest/rest_exporter.go [hunk 7]
```diff
@@ -19,9 +19,7 @@ package rest
 
 import (
 	"sync"
-)
```
Comment (zouyx) [inline protocol/rest/rest_exporter.go:22]: as above

File: remoting/zookeeper/curator_discovery/service_discovery.go [hunk 8]
```diff
@@ -22,18 +22,14 @@ import (
 	"path"
 	"strings"
 	"sync"
-)
 
-import (
```
Comment (zouyx) [inline remoting/zookeeper/curator_discovery/service_discovery.go:27]: recover all import style in this file

File: registry/protocol/protocol.go [hunk 9]
```diff
@@ -21,23 +21,21 @@ import (
 	"context"
 	"strings"
 	"sync"
-)
```
Comment (zouyx) [inline registry/protocol/protocol.go:24]: recover all import style in this file

Comment (YongHaoWu) [reply registry/protocol/protocol.go:24]: @zouyx  Hi, I still not do the changes yet. Will do later.

Comment (YongHaoWu) [reply registry/protocol/protocol.go]: @zouyx  hey, it's done now.

**Review by zouyx — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by gaoxinge — APPROVED** [review]

**Pull request merged**